WordPress/gutenberg
# Paste from Slack: interpret paragraph markup

## What?

Slack gives us some weird markup for "paragraphs" (which are actually created by two line breaks).

```html
<p>test<span class="c-mrkdwn__br" data-stringify-type="paragraph-break" style="box-sizing: inherit; display: block; height: 8px;"></span>test</p>
```

This PR interprets that markup.

## Why?

To make paste from Slack more smooth

|Original|Pasted|
|-|-|
|<img width="140" alt="Screenshot 2022-08-10 at 11 04 28" src="https://user-images.githubusercontent.com/4710635/183861811-31812120-ebed-4314-a000-296737e4d395.png">|<img width="193" alt="Screenshot 2022-08-10 at 11 05 34" src="https://user-images.githubusercontent.com/4710635/183861914-9ccaa2a1-4b31-42d2-a948-4d7967f830d5.png">|

## How?

Adds a new paste rule

## Testing Instructions

Create a message in Slackbot with two lines separated by two line breaks. Send it and copy paste the content in Gutenberg. The two lines should be separate paragraphs.

## Screenshots or screencast
